pages/views/_layouts/content.njk

```diff
@@ -9,6 +9,7 @@
     <div class="content {{ containerCls or 'container' }} {{ 'container-aside' if aside }}">
       {% if title %}
         <h1 class="page-title">
+          {% if icon and not aside %}{% include "static/assets/" + icon %}{% endif %}
```

Comment:
`aside` is not a good criterion to define if we need an icon, 
As a suggestion, try to rename icon assets for samples pages to `preview` then we will not have a conflict with that meta fields